### PR TITLE
[Review] Request from 'aduffeck' @ 'SUSE/machinery/review_150615_fix_typo'

### PR DESCRIPTION
**Pull request by aduffeck**
Please review the following changes:
  * 75b371e Fix typo in progress output

### CODE REVIEW

File: plugins/unmanaged_files/unmanaged_files_inspector.rb [hunk 1]
```diff
@@ -389,7 +389,7 @@ def summary
 
   def show_extraction_progress(count)
     progress = Machinery::pluralize(
-      count, "-> Extracted %d file or tree", "-> Extracted %d files and trees",
+      count, " -> Extracted %d file or tree", "-> Extracted %d files and trees",
```
Comment (cornelius) [inline plugins/unmanaged_files/unmanaged_files_inspector.rb]: Why is there a space for the singular, but no space for the plural form?

There also is a trailing comma, which probably shouldn't be there.

**Comment by aduffeck** [comment]
I fixed the second missing space, the comma is not part of the string, though.

**Comment by thardeck** [comment]
lgtm

**Comment by aduffeck** [comment]
squashed and merged

**Pull request closed without merging**